src/form-question-types.rst

```diff
@@ -1658,33 +1659,151 @@ In the parameters column, write ``max-pixels=`` followed by the desired maximum
 Audio widget
 ----------------
 
-Records audio from an external app.
+type
+  :tc:`audio`
+appearance
+  :tc:`none`
 
-.. note::
+Records audio using the device's microphone or a connected external microphone. By default, an :ref:`external application <external-audio-app>` is used. Starting with Collect v1.29, you can also use :ref:`built-in recording <built-in-audio-recording>`.
 
-  Some Android devices provide a default application for audio recording.
-  Others do not,
-  and the user will need to install an audio recording app.
+.. image:: /img/form-widgets/audio-start.*
+  :alt: The Audio form widget as displayed in the ODK Collect App on an Android phone. The question text is "What does it sound like?" There are two buttons: Record Sound and Choose Sound.
 
-  There are many apps available for this, including:
+.. rubric:: XLSForm
 
-  - `Axet Audio Recorder (open source) <https://play.google.com/store/apps/details?id=com.github.axet.audiorecorder>`_
-  - `RecForge II <https://play.google.com/store/apps/details?id=dje073.android.modernrecforge>`_
+.. csv-table:: survey
+  :header: type, name, label
 
-  Any app that responds to
-  `android.provider.MediaStore.Audio.Media.RECORD_SOUND_ACTION`
-  will be compatible.
+  audio, bird_recording, What does it sound like?
 
+.. tip::
```

Comment:
This felt right in a tip but not 100% sure I've found the right place for it.

Reply:
Seems good to me! 👍